src/styles/card.css

```diff
@@ -0,0 +1,94 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  padding: 32px 16px 0;
+  border-radius: 5px;
+  border: 1px solid var(--light-white);
+  width: var(--card-width);
+  background-color: var(--white);
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
```

Comment:
CHECKLST 8. [STYLES] - Don't hardcode exact height size. Add necessary paddings according to mockup and let content dictate the container size.

Reply:
> CHECKLST 8. [STYLES] - Don't hardcode exact height size. Add necessary paddings according to mockup and let content dictate the container size.

I did this to set the size of the picture but not the block